src/styles/index.scss

```diff
@@ -1,3 +1,10 @@
+@import './variables';
+@import './header';
+@import './stars';
+@import './card';
+
 body {
+  display: block;
```

Comment:
The 'display: block;' property for the 'body' selector is unnecessary as 'block' is the default display value for 'body'. You can safely remove this line to keep the stylesheet clean and avoid redundancy.